README.md

```diff
@@ -28,18 +28,18 @@ This is the C++ API for the Template Task Graph (TTG) programming model for flow
     - `TTG_USE_MADNESS`: selects the MADNESS backend as the default (expert-use only).
 
     For simple programs it is sufficient to include the top-level header file:
-    ```cpp
-    #include <ttg.h>
-    ```
+```cpp
+#include <ttg.h>
+```
```

Comment:
Somethign is wrong with the typesetting here:
![Screenshot from 2022-03-04 16-44-35](https://user-images.githubusercontent.com/10974502/156846016-4ab8a20c-3893-4857-af91-5dc965508498.png)